README.md

```diff
@@ -225,10 +225,17 @@ I'm new in your community,you all are cool^^
 ### Together - we are force!!!!
 
 -----------------------------------------------------------------
-
 ### In force we trust!
 ![cat-force](gif/force.gif)
 
+
+## Hi, Kottans!
+Happy to become a part of this great community!
+### Let's the training begin!!! ;)
+
```

Comment:
One empty line is sufficient to split content into paragraphs. More empty lines do not expand the space between paragraphs and also make source file redundantly longer.